### PR TITLE
Get subscription key and endpoint with env variables

### PR DESCRIPTION
**Pull request by v-jaswel**

### CODE REVIEW

File: Detect.cs [hunk 1]
```diff
@@ -74,11 +92,11 @@ static async Task Main(string[] args)
             // Output languages are defined in the route.
             // For a complete list of options, see API reference.
             // https://docs.microsoft.com/azure/cognitive-services/translator/reference/v3-0-detect
-            string subscriptionKey = "YOUR_TRANSLATOR_TEXT_KEY_GOES_HERE";
-            string host = "https://api.cognitive.microsofttranslator.com";
             string route = "/detect?api-version=3.0";
             string breakSentenceText = @"How are you doing today? The weather is pretty pleasant. Have you been to the movies lately?";
```
Comment (erhopf) [inline Detect.cs]:
```suggestion
            string detectSentenceText = @"How are you doing today? The weather is pretty pleasant. Have you been to the movies lately?";
```

File: Detect.cs [hunk 2]
```diff
@@ -74,11 +92,11 @@ static async Task Main(string[] args)
...
-            await DetectTextRequest(subscriptionKey, host, route, breakSentenceText);
+            await DetectTextRequest(subscription_key, endpoint, route, breakSentenceText);
```
Comment (erhopf) [inline Detect.cs]:
```suggestion
            await DetectTextRequest(subscription_key, endpoint, route, detectSentenceText);
```

File: BreakSentence.cs [hunk 3]
```diff
@@ -26,8 +26,26 @@ public class DetectedLanguage
 
     class Program
     {
+        private const string key_var = "TRANSLATOR_TEXT_SUBSCRIPTION_KEY";
+        private static readonly string subscription_key = Environment.GetEnvironmentVariable(key_var);
```
Comment (erhopf) [inline BreakSentence.cs]: should this be subscriptionKey instead of subscription_key?

File: DictionaryExamples.cs [hunk 4]
```diff
@@ -8,15 +8,29 @@ namespace TranslatorTextQuickStart
...
+            {
+                throw new Exception("Please set/export the environment variable: " + key_var);
+            }
+            if (null == endpoint)
+            {
+                throw new Exception("Please set/export the environment variable: " + endpoint_var);
+            }
+        }
+
         static string path = "/dictionary/examples?api-version=3.0";
```
Comment (erhopf) [inline DictionaryExamples.cs]:
```suggestion
        static string route = "/dictionary/examples?api-version=3.0";
```

This matches the other samples.

File: DictionaryExamples.cs [hunk 5]
```diff
@@ -8,15 +8,29 @@ namespace TranslatorTextQuickStart
...
         // Translate from English to French.
         static string params_ = "&from=en&to=fr";
 
-        static string uri = host + path + params_;
-
-        // NOTE: Replace this example key with a valid subscription key.
-        static string key = "ENTER KEY HERE";
+        static string uri = endpoint + path + params_;
```
Comment (erhopf) [inline DictionaryExamples.cs]:
```suggestion
        static string uri = endpoint + route + params_;
```

File: Detect.cs (hunk 1 above)
Comment (v-jaswel) [reply Detect.cs]: Done.

File: BreakSentence.cs (hunk 3 above)
Comment (v-jaswel) [reply BreakSentence.cs]: Done for all samples.

File: DictionaryExamples.cs (hunk 4 above)
Comment (v-jaswel) [reply DictionaryExamples.cs]: Done.

File: Detect.cs (hunk 2 above)
Comment (v-jaswel) [reply Detect.cs]: Done.

File: DictionaryExamples.cs (hunk 5 above)
Comment (v-jaswel) [reply DictionaryExamples.cs]: Done.

**Comment by v-jaswel** [comment]
subscription_key has been changed to subscriptionKey, host to endpoint, and path to route. I think that covers all of them.

**Comment by erhopf** [comment]
@Jann-Skotdal - Can you merge this PR, please.

**Pull request merged**